packages/foreman/rubygem-http/rubygem-http.spec

```diff
@@ -0,0 +1,109 @@
+# Generated from http-4.1.1.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
This can be dropped
```suggestion
```

Reply:
@ekohl  Should I drop all of the Generated comments from this PR ?